make-dist.sh

```diff
@@ -174,4 +174,5 @@ cat h2o-dist/buildinfo.json \
   | sed -e "s/SUBST_BRANCH_NAME/${BRANCH_NAME}/g" \
   | sed -e "s/SUBST_BUILD_NUMBER/${BUILD_NUMBER}/g" \
   | sed -e "s/SUBST_LAST_COMMIT_HASH/${LAST_COMMIT_HASH}/g" \
+  | sed -e "s/SUBST_SHA256_HASH/${SHA256_HASH}/g" \
```

Comment:
Where do you compute the hash? Why it is not here?

Reply:
I've added it here https://github.com/h2oai/h2o-release-scripts/blob/master/buildh2o3.sh#L111